### PR TITLE
jormungandr: Timestamp to datetime conversion for ubuntu1804

### PR DESCRIPTION
**Pull request by kadhikari**
Please refer to https://jira.kisio.org/browse/NAVP-1072 for details

### CODE REVIEW

**Review by mbounabi — APPROVED** [review]

**Comment by TeXitoi** [comment]
The new test doesn't pass on the old version.

**Comment by kadhikari** [comment]
@TeXitoi i believed have found a solution but does not work. The solution for me is to delete the test i added. What do you suggest?

File: source/jormungandr/jormungandr/interfaces/v1/fields.py [hunk 1]
```diff
@@ -471,7 +471,10 @@ class FieldDateTime(fields.Raw):
 
     def output(self, key, region):
         if 'timezone' in region and key in region:
-            dt = datetime.datetime.utcfromtimestamp(region[key])
+            try:
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/fields.py]: you probably want use a fucntion...

File: source/jormungandr/jormungandr/utils.py [hunk 2]
```diff
@@ -155,7 +156,10 @@ def timestamp_to_datetime(timestamp, tz=None):
     if timestamp >= maxint:
         return None
 
-    dt = datetime.utcfromtimestamp(timestamp)
+    try:
```
Comment (pbougue) [inline source/jormungandr/jormungandr/utils.py]: Can't you mutualize that with the new navitia_utcfromtimestamp ?
Like navitia_utcfromtimestamp is calling this (maybe with tz=UTC) and probably report work from there in here.

**Review by xlqian — APPROVED** [review]

**Review by nonifier — APPROVED** [review]

**Review by TeXitoi — APPROVED** [review]

**Review by TeXitoi — APPROVED** [review]

**Review by TeXitoi — APPROVED** [review]

**Comment by nonifier** [comment]
:1st_place_medal:

**Pull request merged**